### PR TITLE
Performance tests

### PR DESCRIPTION
**Pull request by magdasta**

### CODE REVIEW

**Comment by chfast** [comment]
## [Current coverage](https://codecov.io/github/imapp-pl/golem?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a) is `26.07%`

> Merging **#97** into **develop** will decrease coverage by **-0.58%** as of [`3432819`](https://codecov.io/github/imapp-pl/golem/commit/34328194a7ec8ffb237bdbed056c2179fb2da58a)

``` diff
@@            develop     #97   diff @@
=======================================
  Files           232     245    +13
  Stmts         18977   19669   +692
  Branches       2164    2250    +86
  Methods           0       0       
=======================================
+ Hit            5058    5129    +71
- Partial         192     205    +13
- Missed        13727   14335   +608
```

> Review entire [Coverage Diff](https://codecov.io/github/imapp-pl/golem/compare/52d4dde45d019d74d00b533812adc06697d2dcfa...34328194a7ec8ffb237bdbed056c2179fb2da58a) as of [`3432819`](https://codecov.io/github/imapp-pl/golem/commit/34328194a7ec8ffb237bdbed056c2179fb2da58a)

---

### [Uncovered Suggestions](https://codecov.io/github/imapp-pl/golem/features/suggestions?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a)
1. `+0.15%` via [...ndialogcustomizer.py#235...264](https://codecov.io/github/imapp-pl/golem/gnr/customizers/configurationdialogcustomizer.py?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a#235) 
2. `+0.14%` via [.../task/pbrtgnrtask.py#70...97](https://codecov.io/github/imapp-pl/golem/gnr/task/pbrtgnrtask.py?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a#70) 
3. `+0.13%` via [...ndialogcustomizer.py#296...320](https://codecov.io/github/imapp-pl/golem/gnr/customizers/configurationdialogcustomizer.py?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a#296) 
4. _[See 7 more...](https://codecov.io/github/imapp-pl/golem/features/suggestions?ref=34328194a7ec8ffb237bdbed056c2179fb2da58a)_

> Powered by [Codecov](https://codecov.io). Updated on successful CI builds.

File: testtasks/luxrender/lux_test.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+def lux_performance():
+    '''
+    returns time (in seconds) needed to render the example scene
+    '''
+    output_filename_base = "out"
+    current_dir = get_current_directory(sys.modules[__name__].__file__)
+    directory = os.path.join(current_dir, r'lux_task')
+    output_path = os.path.join(directory, output_filename_base)
+    checksum = get_hash_of_dir(directory)
```
Comment (badb) [inline testtasks/luxrender/lux_test.py:21]: I'm getting different hash here, therefore test result is always -1. I don't now way, but during debug there are two files in lux_task dir: .lxs and lxs~, maybe that is the reason... Artur or Paweł can you check if you have the same problem or is it problem with my system?

File: golem/appconfig.py [hunk 2]
```diff
@@ -64,6 +66,30 @@ def read_estimated_performance(cls):
...
+    
+    @classmethod
+    def read_estimated_blender_performance(cls):
+        estm_file = SimpleEnv.env_file_name(ESTM_BLENDER_FILENAME)
+        res = 0
+        if os.path.isfile(estm_file):
+            try:
+                with open(estm_file, 'r') as file_:
+                    res = "{0:.1f}".format(float(file_.read()))
+            except:
```
Comment (badb) [inline golem/appconfig.py:98]: I know that there are [too broad exception clause problem](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) in many places in the code, but I'm planning to [fix it](https://github.com/imapp-pl/golem/issues/34). 
So can you please add there IOError and ValueError?

File: golem/appconfig.py [hunk 3]
```diff
@@ -249,6 +286,18 @@ def get_estimated_performance(self):
         except:
             return float(ESTIMATED_DEFAULT)
 
+    def get_estimated_lux_performance(self):
+        try:
+            return float(self._cfg.get_node_config().getEstimatedLuxPerformance())
+        except:
```
Comment (badb) [inline golem/appconfig.py]: Same issue as in a comment above.

File: testtasks/dir_hash_builder.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+                while 1:
+                    # Read file in as little chunks
+                    buf = f1.read(4096)
+                    if not buf: 
+                        break
+                    SHAhash.update(hashlib.sha1(buf).hexdigest())
+                f1.close()
+
+    except:
```
Comment (badb) [inline testtasks/dir_hash_builder.py:46]: This except can stay as it is.

File: testtasks/luxrender/lux_task/schoolcorridor.lxs~ [hunk 5]
```diff
@@ -0,0 +1,90 @@
+# Main Scene File
```
Comment (chfast) [inline testtasks/luxrender/lux_task/schoolcorridor.lxs~:1]: I suspect it is a temporary file added by accident.

**Pull request merged**